pyproject.toml

```diff
@@ -65,6 +66,14 @@ docs = [
     "IPython",
     "tensorflow==2.16.*",
 ]
+hpo = [
+    "bayesian-optimization==2.0.0",
+    "hyperopt==0.2.7",
+    "ConfigSpace==1.2.1",
+    "hpbandster==0.7.4",
+    "scipy==1.14.1",
+    "GPy==1.13.2"
```

Comment:
I would avoid fixing the patch version when using `==`. Example: `hyperopt==0.2.*`

Reply:
Okay I'll change it

Reply:
if you do `>=` it's fine, but with `==` you are not allowing new patches to the same version. You can fix this in cases you see `==` and the patch version

Reply:
Added!